k9mail/src/main/java/com/fsck/k9/K9.java

```diff
@@ -255,6 +256,9 @@
     private static boolean sMessageViewCopyActionVisible = false;
     private static boolean sMessageViewSpamActionVisible = false;
 
+    private static boolean sUseSocksProxy = false;
+    private static String sSocksProxyHost = "";
+    private static int sSocksProxyPort = 0;
 
```

Comment:
It's a pity that the design has led to this class filling up with lots of constants. 

Would not make sense to just have

```private static ProxySettings sSocksProxySettings``` here instead.